Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -837,6 +838,34 @@ public void onError(RequestError requestError) {
     }
   }
 
+  public List<TypeHierarchyItem> search_getTypeHierarchy(@NotNull final String _filePath, final int offset, final boolean superOnly) {
```

Comment:
Yes, I'm working on the PR that will include the new DAS client, which supports this flag.
Support of it in DAS was committed yesterday.

This will fix bugs with jumping to subclasses.